microproxy/cert.py

```diff
@@ -58,3 +58,13 @@ def get_cert_from_cache(self, common_name):
             return self.certs_cache[common_name]
         except KeyError:
             return None
+
+_cert_store = None
+
+
+def get_cert_store(config=None):
+    global _cert_store
+    if _cert_store:
+        return _cert_store
+    _cert_store = CertStore(config)
+    return _cert_store
```

Comment:
yeah, I agree. That's why in the next commit, I haved move the get_cert_store into server_state and init_server_state(config) to initialize the state, and get_server_state() to get the related data.